pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -45,6 +46,10 @@ func checkRunningInsideWSL2() error {
 }
 
 func checkVirtualizationEnabled() error {
+	if runtime.GOARCH == "arm64" {
+		logging.Debug("Ignoring virtualization check for arm64")
```

Comment:
maybe make this a `logging.Warn` so users see it also when not running with `--log-level debug`?

Reply:
This  is expected behaviour, this is not an error or something problematic so I'd keep this quiet, ie debug.

Reply:
does it mean that all arm cpus support virtualization?

Reply:
I think all aarch64 machines do, yes. While writing this, I'm thinking this likely to be wrong for aarch64 virtual machines, at least Apple cpus did not have nested virt support until recently, and the hypervisor has to support it as well.

Reply:
there's the `systemd-detect-virt` which can detect if running inside a container or virtualization, but can't say if nested virt is supported